src/AssemblySharedInfoGenerator/AssemblySharedInfo.tt

```diff
@@ -68,9 +68,9 @@ using System.Runtime.InteropServices;
 [assembly: AssemblyFileVersion("<#= this.MajorVersion #>.<#= this.MinorVersion #>.<#= this.BuildNumber #>.<#= this.RevisionNumber #>")]
 <#+
 int MajorVersion = 2;
-int MinorVersion = 15;
+int MinorVersion = 16;
 int BuildNumber = 0;
 // The datetime baseline we choose using this algorithm will affect build number and all nuget packages uploaded
 // Please only change when major or minor version got incremented
-int RevisionNumber = ((int)(DateTime.UtcNow - new DateTime(2021,1,1)).TotalDays)*10+(int)DateTime.UtcNow.Hour;
```

Comment:
Per discussion on slack: https://autodesk.slack.com/archives/C01C8GGT9EY/p1654675092409759, reverting back to single-digit to represent hour for now